packages/dds/merge-tree/src/client.ts

```diff
@@ -379,7 +390,7 @@ export class Client {
                         UniversalSequenceNumber,
                         false,
                         { op: removeOp });
-                } else {
+                } else /* if (op.type === MergeTreeDeltaType.ANNOTATE) */ {
```

Comment:
nit: I feel like `} else /* op.type === MergeTreeDeltaType.ANNOTATE */ {` makes it more clear this is a reminder (rather than, say, a leftover experimental conditional).